src/ck-ldb/CentralLB.C

```diff
@@ -224,8 +209,8 @@ void CentralLB::ProcessAtSync()
     // reduction to get total number of objects and comm
     // so that processor 0 can pre-allocate load balancing database
   int counts[2];
-  counts[0] = theLbdb->GetObjDataSz();
-  counts[1] = theLbdb->GetCommDataSz();
+  counts[0] = lbmgr->GetObjDataSz();
```

Comment:
Aren't these also things that are part of the LBDatabase?

Reply:
Internally LBManager calls lbdb_obj->GetObjDataSz(). We had discussed that with this interface, in case we change LBDatabase later, clients wouldn't need to change anything and could continue to use lbmgr directly.